CHANGES.txt

```diff
@@ -22,6 +22,7 @@ Other
   promising support for versions supported by the Python core team,
   Ubuntu LTS, or Debian stable, and taking into consideration
   Debian oldstable and PyPy.
+* Replace `polyfill.io` with `cdnjs.cloudflare.com/polyfill`.
```

Comment:
Please note that this file is using reStructuredText, so you need to use double backticks:
```suggestion
* Replace ``polyfill.io`` with ``cdnjs.cloudflare.com/polyfill``.
```